README.md

```diff
@@ -32,6 +32,7 @@ Event emitter allows you to produce events using REST.
 ```shell
 curl --location --request POST 'http://localhost:8079/events/raw' \
 --header 'Content-Type: application/json' \
+--header 'customerCategory: RETAIL' \
```

Comment:
Reminder that in the "official" solution we might have different headers for HTTP and Async, for example:

- http: `X-LOB`
- async: `bbLineOfBusiness`

So the ideal situation would be to propagate the header for the event with a different name of the one received in the request.

This could be configurable, like:

```
backbase:
  event-emitter:
    headers-lob:
      http: X-LOB
      async: bbLineOfBusiness
```

If this will take time we can leave as is but having in mind that during the tests we might need to pass `bbLineOfBusiness: retail`.


cc @afsinka-backbase @amorimluzia

Reply:
Updated the commit. Used a bit different config:
```
backbase:
  event-emitter:
    custom-header-pairs:
      - http: X-LOB
        event: bbLineOfBusiness
```